samples/samples-outofproc/Microsoft.Azure.WebJobs.Extensions.Sql.SamplesOutOfProc.csproj

```diff
@@ -19,8 +19,10 @@
       Include="..\..\Worker.Extensions.Sql\src\Microsoft.Azure.Functions.Worker.Extensions.Sql.csproj" />
     <!-- We want to build the extension project first since this project references the package
     produced by building that. But we don't want it as an actual runtime depdendency -->
-    <ProjectReference Include="..\..\src\Microsoft.Azure.WebJobs.Extensions.Sql.csproj"
-      ReferenceOutputAssembly="false" />
+    <ProjectReference Include="..\..\src\Microsoft.Azure.WebJobs.Extensions.Sql.csproj">
+        <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
+        <Private>false</Private>
```

Comment:
Technically we shouldn't need to do this since the extensions project doesn't generate other output files, but adding it doesn't hurt and will prevent issues later on if we do add something.